lib/actuators/include/DCMotor.h

```diff
@@ -1,14 +1,11 @@
 #pragma once
 
-#ifndef MOTOR_DEFAULT_FREQUENCY_HZ
-#define MOTOR_DEFAULT_FREQUENCY_HZ 244  // 10 kHz
-#endif
+#include "Actuator.h"
 
-#include "mbed.h"
-
-// Interface to control a standard DC motor with an H-bridge using a PwmOut and 2 DigitalOuts
+class DCMotor : public Actuator {
+ protected:
+  static constexpr int MOTOR_DEFAULT_FREQUENCY_HZ = 1000;  // TODO
```

Comment:
why is this static and why it used to say 244?

Reply:
I'm not sure actually, I copied it over from the existing Motor.h code

Reply:
Before the line was `#define MOTOR_DEFAULT_FREQUENCY_HZ 244  // 10 kHz` and the constructor comment was `@param freqInHz  Output PWM frequency, default 1kHz`, which are all contradicting

Reply:
hmm can u ask one of the older members on the team? maybe Alex?